### ISSUE TITLE
Track the cleaning time for the rooms.

### ISSUE THREAD

**Issue by aferrato** [humbertogontijo/homeassistant-roborock#573]
## Is your feature request related to a problem? Please describe.

It would be helpful to have the ability to track the cleaning time for the different rooms: start time, end time, and the cleaning method used. I would like to automate house cleaning based on when the rooms were last cleaned. Thank you!

## Describe the solution you'd like
Create an entity in Home Assistant for each room with the time of the last cleaning. Additionally, add attributes for the start time, end time, total time of last cleaning, and cleaning method.

## Describe alternatives you've considered
I'm trying to create in Node-RED the capability to track the last cleaning based on when the current room sensor changes state and the robot is in the cleaning phase.

**Comment by arne182** [comment]
I wonder if a blueprint could do this. I have helpers and automation for every room. If cleaning  and current room is xxx and it has been x minutes then set time of helper for that room

**Comment by aferrato** [comment]
> I wonder if a blueprint could do this. I have helpers and automation for every room. If cleaning and current room is xxx and it has been x minutes then set time of helper for that room

Yes, I was also thinking about something like that. But if I'm not mistaken, the robot is "in cleaning" even while it's moving to reach the room to be cleaned, and this could create issues with the record.

**Comment by arne182** [comment]
Yes that is why I have set it so that it must be at least 5-10 minutes in a room before setting the helper. Not perfect but it works for what I need it for.
